### PR TITLE
XHTML validity (team 3)

### PR DESCRIPTION
**Pull request by ghost**
Fixed invalid things in XHTML. (#82)

### CODE REVIEW

File: piplmesh/frontend/templates/header.html [hunk 1]
```diff
@@ -7,15 +7,16 @@
...
         </div>
         <div id="search">
-            <div id="cse_search_form"></div>
-            <script src="https://www.google.com/jsapi" type="text/javascript"></script>
-            <script type="text/javascript">
-                var language_code = '{{ LANGUAGE_CODE }}';
-                var search_engine_unique_id = '{{ SEARCH_ENGINE_UNIQUE_ID }}';
-                var search_page_url = '{% url search %}';
-            </script>
+            <div id="cse_search_form">
```
Comment (ghost) [inline piplmesh/frontend/templates/header.html:10]: Please put this line back as it was, it was like that for a reason.

**Comment by mitar** [comment]
In registration form I am getting some warnings.

File: piplmesh/frontend/templates/header.html (hunk 1 above)
Comment (ghost) [inline piplmesh/frontend/templates/header.html:10]: Ok, I'll change this back. Just out of curiosity - what's the reason?

Comment (mitar) [inline piplmesh/frontend/templates/header.html:10]: Probably placeholder for the content which is then filled from JavaScript.

Comment (ghost) [inline piplmesh/frontend/templates/header.html:10]: So there is no difference, because I just moved the <//div> tag  at the end of JS code.

**Comment by ghost** [comment]
I don't really know how to fix these warnings. The problem is that the label for Gender "needs" 2 different "for" atributes, because of id_gender_0 and id_gender_1 "id" atributes of radio buttons.

File: piplmesh/frontend/templates/header.html (hunk 1 above)
Comment (mitar) [inline piplmesh/frontend/templates/header.html:10]: Exactly this is the difference. Once it is an empty placeholder where JavaScript code replaces the content with new one. In your case there is something inside which get replaced with new content and we lose JavaScript. I don't know what browser does in such case, but I do not want to disk if it is not necessary.

Comment (ghost) [inline piplmesh/frontend/templates/header.html:10]: Ok, changing  back.

**Comment by mitar** [comment]
Read here:

http://www.w3.org/TR/WCAG20-TECHS/H44.html#H44-ex3
http://www.w3.org/TR/WCAG20-TECHS/H71.html

So you will have to change template that for multiple fields it uses fieldset instead of label. Or: maybe you could always use fieldset for each row and labels only for those fields consisted of multiple inputs.

**Comment by mitar** [comment]
Can you also fix the warnings please?

**Pull request merged**